### ISSUE TITLE
False positive with some.package.typing.SomeType

### ISSUE THREAD

**Issue by scop** [asottile/flake8-typing-imports#11]
Types imported from a package path that ends with `.typing` (not just the top-level `typing`) result in false positives.

```
$ find
.
./x
./x/typing.py
./x/__init__.py

$ cat x/typing.py 
ExampleType = str

$ cat x/__init__.py 
from .typing import ExampleType


def foo(ex: ExampleType):
    pass

$ flake8 x
x/__init__.py:1:1: TYP001 guard import by `if False:  # TYPE_CHECKING`: ExampleType (not in 3.5.0, 3.5.1, 3.5.2, 3.5.3, 3.5.4, 3.5.5, 3.5.6, 3.5.7, 3.6.0, 3.6.1, 3.6.2, 3.6.3, 3.6.4, 3.6.5, 3.6.6, 3.6.7, 3.6.8, 3.6.9, 3.7.0, 3.7.1, 3.7.2, 3.7.3, 3.7.4)
```

If `ExampleType` here is imported from somewhere else, e.g. `x.blah`, no such problem.

**Comment by asottile** [comment]
thanks for the report! this is fixed in v1.5.1
